packages/docs/src/components/layout.js

```diff
@@ -6,30 +6,24 @@ import {
   Main,
   Box,
   Container,
-  useColorMode,
+  useColorMode
 } from 'theme-ui'
 import { useState, useRef } from 'react'
 import { Global } from '@emotion/core'
+import { Helmet } from 'react-helmet'
+import pkg from 'theme-ui/package.json'
 
 import SkipLink from './skip-link'
 import Header from './header'
 import Footer from './footer'
 import Sidebar from './sidebar'
 import Pagination from './pagination'
-import MenuButton from './menu-button'
-import NavLink from './nav-link'
-import Button from './button'
 
-const modes = [
-  'light',
-  'dark',
-  'deep',
-  'swiss',
-]
+const modes = ['light', 'dark', 'deep', 'swiss']
 
 export default props => {
-  const [ menuOpen, setMenuOpen ] = useState(false)
-  const [ mode, setMode ] = useColorMode()
+  const [menuOpen, setMenuOpen] = useState(false)
+  const [mode, setMode] = useColorMode()
```

Comment:
This hook could go into the Header component now, couldn't it?

Reply:
Yes it can 👍